bgpd/bgp_evpn.c

```diff
@@ -3224,6 +3226,40 @@ static void bgp_evpn_handle_export_rt_change_for_vrf(struct bgp *bgp_vrf)
 		update_routes_for_vni(bgp_def, vpn);
 }
 
+/*
+ * Handle autort change for a given VNI.
+ */
+static void update_autort_vni(struct hash_backet *backet, struct bgp *bgp)
+{
+	struct bgpevpn *vpn = backet->data;
+	struct listnode *node, *nnode;
+	struct ecommunity *ecom;
+	if (!vpn) {
```

Comment:
We try to put an empty line here to separate out the variable declarations and actual code.